packages/office-ui-fabric-react/src/components/HoverCard/ExpandingCard.tsx

```diff
@@ -68,6 +68,7 @@ export class ExpandingCard extends BaseComponent<IExpandingCardProps, IExpanding
       theme,
       styles: customStyles,
       compactCardHeight,
+      directionalHintFixed = true,
```

Comment:
Put the defaults in `public static defaultProps`

Should we also not expose this prop in HoverCard?